### ISSUE TITLE
Sincere Question about setting up the Vote Shop

### ISSUE THREAD

**Issue by ghost** [BenCodez/VotingPlugin#478]
Dear Ben,

Thank you for this plugin, it is quite useful. I have, however, run into a kind of road block. Forgive me for asking, because its probably a simple question that you might even have answered in your wiki (though I could not find it), but how exactly do you set up the vote shop? I have voting functioning well and vote points properly buying the example item (diamond for 3 vote points) in the shop, but I have no idea how to add or remove items. I have tried messing around in the configs, but due to my ignorance of the knowledge necessary to be a server administrator, I could not find what I needed.

/vote shop works fine

TL;DR: I need to know how to add/delete items from the /Vote Shop

Thank you for your time, and I hope a solution can be found quickly. If I wasted your time in asking this when it has already been answered elsewhere, I apologize.

Sincerely, 
Zodd_Noward

**Comment by BenCodez** [comment]
Shop:
      # Identifier
      # Used in placeholders for formats
      Item1:
        # data
      Item2:
        # data
      Item:
        # data
        
Something like that.

**Comment by ghost** [comment]
And that goes in which config file? config.yml or votesites.yml?
I'm assuming config.yml

**Comment by BenCodez** [comment]
Config.yml

**Comment by ghost** [comment]
![screen shot 2018-12-28 at 6 09 32 pm](https://user-images.githubusercontent.com/28005422/50531194-9838aa00-0acb-11e9-81e0-65e15e2e1347.png)
Ah, I found it!

**Comment by ghost** [comment]
Thank you very much. To make more than one, I'll just copy this formatting, right?

**Comment by BenCodez** [comment]
Yep

**Comment by ghost** [comment]
Ok, thanks again! I will now close this issue.
